### DISCUSSION TITLE
Payment starter project: Stripe webhook handler errors (400)

### DISCUSSION THREAD

**Discussion by angadveeersb** [vercel/community#3198] [Help] [answered]
### Summary

I've followed the instructions in the README of the project repo carefully and yet I face this issue.
## Problem Overview
- Stripe is making successful requests to Vercel's Serverless Functions as there are a few successful 200 requests
- But mostly, important events like price.created or product.created are throwing error 400 and hence, products are not being created in the website automatically.

## Help Required
- Could you please try out the webhook and the starter project for yourself? (if you are working on this project now that would be great!) 
- Instead of fixing the event handling for product/price creation, is there a way to hardcode my own prices and products and link them to their respective price IDs in Stripe and how can I do this?

I'm a beginner at web development so I need more guidance. Thank you so much for reading through and I appreciate your help!

**Comment by amyegan** [comment]
Sorry you're having trouble with this, @angadveeersb. Are you using the [Stripe Subscription Starter](https://vercel.com/templates/next.js/subscription-starter) template, or something different? I want to be sure I investigate the right thing 😅

**Reply by angadveeersb** [reply]
thanks for your reply, I believe there has been a commit that changed the README file which caused me to follow the updated one and miss an important step.

**Reply by angadveeersb** [reply]
also, yes i am referring to the [Stripe Subscription Starter](https://vercel.com/templates/next.js/subscription-starter) template

**Comment by angadveeersb** [comment, marked as answer]
Friends, I solved it. Just do the following. Somehow there has been a commit around 3 hours ago that removed this section from the Deploy button section 

## Run `schema.sql`

+ From your Github fork, copy the code from `schema.sql`. In your Supabase project, navigate to the SQL editor, click `New Query`, paste the code, and run the code. This will create the necessary tables and RLS policies in your Supabase database.
